include/CLI/TypeTools.hpp

```diff
@@ -610,9 +639,10 @@ template <typename T> struct classify_object<T, typename std::enable_if<std::is_
 
 /// String and similar direct assignment
 template <typename T>
-struct classify_object<T,
-                       typename std::enable_if<!std::is_floating_point<T>::value && !std::is_integral<T>::value &&
-                                               std::is_assignable<T &, std::string>::value>::type> {
+struct classify_object<
+    T,
+    typename std::enable_if<!std::is_floating_point<T>::value && !std::is_integral<T>::value &&
+                            std::is_assignable<T &, std::string>::value && !force_widen<T>::value>::type> {
```

Comment:
Here basically, I make it so that an object, in the event that it is both assignable and/or constructible from `std::string `AND from `std::wstring`, will be classified as `wstring_assignable` / `wstring_constructible` (the wide one) if it's `force_widen`